Error using CRA with Chakra UI JS template on Windows 10

# 🐛 Bug report

> Error using CRA with Chakra UI JS template on Windows 10

## 💥 Steps to reproduce

run `npx create-react-app my-app --template @chakra-ui`

## 🧐 Expected behavior

> To be able to start a new CRA project with Chakra UI

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Chakra UI        |     1.0       |
| React               |   17.0.1    |
| Node               |   15.4.0    |
| npm                 |   7.0.15    |
| OS                   |   Windows 10    |

## 📝 Additional information
- Error log:
 ```
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   peer react@"17.0.1" from react-dom@17.0.1
npm WARN   node_modules/react-dom
npm WARN     react-dom@"^17.0.1" from the root project
npm WARN     4 more (framer-motion, @testing-library/react, ...)
npm WARN   52 more (the root project, @chakra-ui/react, @emotion/react, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from @reach/auto-id@0.11.0
npm WARN node_modules/@chakra-ui/hooks/node_modules/@reach/auto-id
npm WARN   @reach/auto-id@"0.11.0" from @chakra-ui/hooks@1.0.1
npm WARN   node_modules/@chakra-ui/hooks
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   peer react@"17.0.1" from react-dom@17.0.1
npm WARN   node_modules/react-dom
npm WARN     react-dom@"^17.0.1" from the root project
npm WARN     4 more (framer-motion, @testing-library/react, ...)
npm WARN   52 more (the root project, @chakra-ui/react, @emotion/react, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.14.0" from react-dom@16.14.0
npm WARN node_modules/@chakra-ui/hooks/node_modules/react-dom
npm WARN   peer react-dom@"^16.8.0" from @reach/auto-id@0.11.0
npm WARN   node_modules/@chakra-ui/hooks/node_modules/@reach/auto-id
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   peer react@"17.0.1" from react-dom@17.0.1
npm WARN   node_modules/react-dom
npm WARN     react-dom@"^17.0.1" from the root project
npm WARN     4 more (framer-motion, @testing-library/react, ...)
npm WARN   52 more (the root project, @chakra-ui/react, @emotion/react, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from react-remove-scroll@2.4.0
npm WARN node_modules/react-remove-scroll
npm WARN   react-remove-scroll@"2.4.0" from @chakra-ui/modal@1.2.1
npm WARN   node_modules/@chakra-ui/modal
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   peer react@"17.0.1" from react-dom@17.0.1
npm WARN   node_modules/react-dom
npm WARN     react-dom@"^17.0.1" from the root project
npm WARN     4 more (framer-motion, @testing-library/react, ...)
npm WARN   52 more (the root project, @chakra-ui/react, @emotion/react, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from react-focus-lock@2.4.1
npm WARN node_modules/react-focus-lock
npm WARN   react-focus-lock@"2.4.1" from @chakra-ui/focus-lock@1.0.1
npm WARN   node_modules/@chakra-ui/focus-lock
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   peer react@"17.0.1" from react-dom@17.0.1
npm WARN   node_modules/react-dom
npm WARN     react-dom@"^17.0.1" from the root project
npm WARN     4 more (framer-motion, @testing-library/react, ...)
npm WARN   52 more (the root project, @chakra-ui/react, @emotion/react, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from @reach/alert@0.11.0
npm WARN node_modules/@reach/alert
npm WARN   @reach/alert@"0.11.0" from @chakra-ui/toast@1.1.1
npm WARN   node_modules/@chakra-ui/toast
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react-dom@17.0.1
npm WARN node_modules/react-dom
npm WARN   react-dom@"^17.0.1" from the root project
npm WARN   4 more (framer-motion, @testing-library/react, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react-dom@"^16.8.0" from @reach/alert@0.11.0
npm WARN node_modules/@reach/alert
npm WARN   @reach/alert@"0.11.0" from @chakra-ui/toast@1.1.1
npm WARN   node_modules/@chakra-ui/toast
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   peer react@"17.0.1" from react-dom@17.0.1
npm ERR!   node_modules/react-dom
npm ERR!     react-dom@"^17.0.1" from the root project
npm ERR!     peer react-dom@">=16.8" from framer-motion@2.9.5
npm ERR!     node_modules/framer-motion
npm ERR!       framer-motion@"^2.9.4" from the root project
npm ERR!       8 more (@chakra-ui/react, @chakra-ui/checkbox, ...)
npm ERR!     3 more (@testing-library/react, @chakra-ui/portal, @chakra-ui/toast)
npm ERR!   react@"^17.0.1" from the root project
npm ERR!   51 more (@chakra-ui/react, @emotion/react, @emotion/styled, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from @reach/utils@0.11.0
npm ERR! node_modules/@reach/alert/node_modules/@reach/utils
npm ERR!   @reach/utils@"0.11.0" from @reach/alert@0.11.0
npm ERR!   node_modules/@reach/alert
npm ERR!     @reach/alert@"0.11.0" from @chakra-ui/toast@1.1.1
npm ERR!     node_modules/@chakra-ui/toast
npm ERR!       @chakra-ui/toast@"1.1.1" from @chakra-ui/react@1.0.3
npm ERR!       node_modules/@chakra-ui/react
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\ugoro\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ugoro\AppData\Local\npm-cache\_logs\2020-12-17T19_27_41_805Z-debug.log
`npm install --save @chakra-ui/react@^1.0.0 @emotion/react@^11.0.0 @emotion/styled@^11.0.0 react-icons@^3.0.0 framer-motion@^2.9.4 @testing-library/jest-dom@^5.9.0 @testing-library/react@^10.2.1 @testing-library/user-event@^12.0.2 web-vitals@^0.2.2` failed
```
- Creating a new CRA project without Chakra UI template or with other templates (tested with redux toolkit) works perfectly
- Tried creating a new CRA project with no template and then running `npm i @chakra-ui/react @emotion/react @emotion/styled framer-motion --legacy-peer-deps` and it seems to have worked

--- a/.changeset/blue-games-join.md
+++ b/.changeset/blue-games-join.md
@@ -1,0 +1,7 @@
+---
+"@chakra-ui/focus-lock": minor
+"@chakra-ui/modal": minor
+---
+
+Upgrade to react-remove-scroll@2.4.1 and react-focus-lock@2.5.0 to fix React 17
+peer dependencies compatibility

--- a/packages/focus-lock/package.json
+++ b/packages/focus-lock/package.json
@@ -52,7 +52,7 @@
   },
   "dependencies": {
     "@chakra-ui/utils": "1.2.0",
-    "react-focus-lock": "2.4.1"
+    "react-focus-lock": "2.5.0"
   },
   "peerDependencies": {
     "react": ">=16.8.6"

--- a/packages/modal/package.json
+++ b/packages/modal/package.json
@@ -66,7 +66,7 @@
     "@chakra-ui/transition": "1.0.8",
     "@chakra-ui/utils": "1.2.0",
     "aria-hidden": "^1.1.1",
-    "react-remove-scroll": "2.4.0"
+    "react-remove-scroll": "2.4.1"
   },
   "devDependencies": {
     "@chakra-ui/system": "1.3.0",

